app/statistics/template.hbs

```diff
@@ -214,7 +214,7 @@
         @type={{this.type}}
         @missingParams={{this.missingParams}}
         @ordering={{this.ordering}}
-        @on-ordering-change={{fn (mut this.ordering)}}
+        @onOrderingChange={{fn (mut this.ordering)}}
```

Comment:
This is the actual fix of the "sorting on header click is not working".

Reply:
Sweet